.github/workflows/dev-docs.yml

```diff
@@ -21,6 +21,8 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: 12
+      - name: Install Graphviz
```

Comment:
Not related to this line, but do we need to have `actions/setup-java@v1`, at least to specify the version?

Reply:
It should work without explicit installation as java 11 is available in ubuntu-latest, but adding it to specify the version is fine as well. I have updated it accordingly.

Reply:
> It should work without explicit installation as java 11 is available in ubuntu-latest

Making this assumption means we're relying on internal settings of the machine, which may change in a way that can break the build.